tests/k8s/tests/00-gsg-test.bash

```diff
@@ -109,10 +107,9 @@ if [[ "${RETURN//$'\n'}" != "200" ]]; then
 fi
 
 echo "----- creating L7-aware policy -----"
-kubectl create -f "${MINIKUBE}/l3_l4_l7_policy.yaml"
+k8s_apply_policy kube-system "${MINIKUBE}/l3_l4_l7_policy.yaml"
 
-echo "----- Waiting for endpoints to get into 'ready' state -----"
-wait_all_k8s_regenerated 5
+k8s_wait_for_revsion $NEXT_REV $NAMESPACE $CILIUM_POD_1 $CILIUM_POD_2
```

Comment:
this should be `k8s_wait_for_revision` - where is this function declared? The only place I see this function referenced is [here](https://github.com/cilium/cilium/pull/1196/files#diff-eedb878922c5a9cab1ca990235a60a1dR372), and it's a comment in `helpers.bash`. Is this intended?